docs/builds/guides/migration/migration-to-27.0.0.md

```diff
@@ -13,14 +13,86 @@ Listed below are the most important changes that require your attention when upg
 
 The {@link module:engine/view/document~Document#event:clipboardInput `view.Document#clipboardInput`} and {@link module:clipboard/clipboardpipeline~ClipboardPipeline#event:inputTransformation `ClipboardPipeline#inputTransformation`} events should not be fired or stopped in your feature code. The `data.content` property should be assigned to override the default content instead. You can stop this event only if you want to completely disable pasting/dropping of some specific content.
 
 You can read about the whole input pipeline in details in {@link framework/guides/deep-dive/clipboard#input-pipeline clipboard pipeline guide}.
+
+## The `view.Document` event bubbling
+
+In v27.0.0 we introduced bubbling of the {@link module:engine/view/document~Document `view.Document`} events, similar to how bubbling works in the DOM. That allowed us to reprioritize many listeners that previously had to rely on the `priority` property. However, it means that existing listeners that use priorities may now be executed at a wrong time (in the different order). These listeners should be reviewed in terms of when they should be executed (in what context/element/phase).
+
+Read more about bubbling events in the {@link framework/guides/deep-dive/event-system#bubbling-events event system guide}.
+
+### The `delete` event
+
+Previously, the {@link module:engine/view/document~Document#event:delete `delete`} event was handled by different features on the different priority levels to, for example, ensure the precedence of the list item over the block quote that is wrapping it. From v27.0.0 on, this precedence is handled by the events bubbling over the view document tree. Listeners registered for the view elements deeper in the view tree are now triggered before listeners for elements closer to the {@link module:engine/view/rooteditableelement~RootEditableElement root element}.
+
+Let's take a look at the list of `delete` listeners across core editor features and their {@link module:utils/priorities~PriorityString priorities}:
+
+| **Feature**        | **Priority before v27** | **Event context from v27** |
+| ---                | ---                     | ---                        |
+| List               | High + 10               | `li` @ Normal              |
+| BlockQuote         | High + 5                | `blockquote` @ Normal      |
+| Widget type around | High + 1                | *isWidget* @ Normal        |
+| Widget             | High                    | `$root` @ Normal           |
+| Delete             | Normal                  | `$document` @ Low          |
+
+Looking at this table, even if your listener was listening on the `highest` priority, it will be triggered just before the last handler that is listening on the `$document` at the `low` priority because the `$document` is the {@link framework/guides/deep-dive/event-system#listening-to-bubbling-events default context} for registering listeners.
```

Comment:
TBH, I was looking at this table and I don't get it so I guess it's rather unlikely developers reading this will get it. I think this paragraph needs to be rephrased.